specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-10-01/machineLearningServices.json

```diff
@@ -4937,6 +5022,126 @@
         "modelAsString": true
       }
     },
+    "TriggerType": {
+      "type": "string",
+      "description": "The schedule trigger type.",
+      "enum": [
+        "Recurrence",
+        "Cron"
+      ],
+      "x-ms-enum": {
+        "name": "TriggerType",
+        "modelAsString": true
+      }
+    },
+    "Recurrence": {
+      "type": "object",
+      "description": "The workflow trigger recurrence for ComputeStartStop schedule type.",
+      "properties": {
+        "frequency": {
+          "description": "[Required] The frequency to trigger schedule.",
+          "$ref": "#/definitions/RecurrenceFrequency"
+        },
+        "interval": {
+          "description": "[Required] Specifies schedule interval in conjunction with frequency",
```

Comment:
>[Required] Specifies schedule interval in conjunction with frequency [](http://example.com/codeflow?start=26&length=68)

The names frequency and interval are a little confusing. I think interval and intervalUnits would a little clearer.

Reply:
The naming of parameters here conforms to the way that Logic App Team defines recurrrence. See below link: 
https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-workflow-actions-triggers#triggers-overview